cargo-dist/tests/snapshots/markdown-help.snap

```diff
@@ -155,8 +155,8 @@ Automatically accept all recommended/default values
 
 This is equivalent to just mashing ENTER over and over during the interactive prompts.
 
-#### `--no-generate`
-Don't automatically invoke 'cargo dist generate' at the end
+#### `--skip-generate`
```

Comment:
Are we okay with renaming this commandline option? Consistency is good, just want to avoid confusing people.

Reply:
There's aliases for the old name (alas not exposed in docs).